discordgsm/games.csv

```diff
@@ -21,6 +21,8 @@ armareforger,Arma Reforger (2022),source,port=2001;port_query=17777
 asa,Ark: Survival Ascended (2023),asa,port=7777
 assettocorsa,Assetto Corsa (2014),assettocorsa,port=9610
 atlas,Atlas (2018),source,port=5761;port_query_offset=51800
+astrocolony,Astro Colony (2022),source,port_query=27015
+avorion,Avorion (2017),source,port=40017,port_query_offset=2
```

Comment:
should be `port=40017;port_query_offset=2`

Reply:
ah good catch, fixed